src/main/java/io/stargate/sgv2/jsonapi/config/constants/DocumentConstants.java

```diff
@@ -27,9 +27,6 @@ interface Fields {
     /** Document field name that will have text value for which vectorize method in called */
     String VECTOR_EMBEDDING_TEXT_FIELD = "$vectorize";
 
-    /** Key for vector function name definition in cql index. */
-    String VECTOR_INDEX_FUNCTION_NAME = "similarity_function";
-
```

Comment:
Duplicated with `VectorConstants`. I think it's better in there, so I remove the one in here